clang/test/CAS/libclang-prune-data.c

```diff
@@ -0,0 +1,61 @@
+// Tests that the CAS directory storage can be limited via libclang APIs.
+// The test depends on internal details of the CAS directory structure.
+
+// RUN: rm -rf %t && mkdir -p %t
+
+// RUN: %clang -cc1depscan -fdepscan=inline -fdepscan-include-tree -o %t/t.rsp -cc1-args \
+// RUN:   -cc1 -triple x86_64-apple-macos12 -fcas-path %t/cas -emit-obj %s -o %t/output.o
+// RUN: %clang @%t/t.rsp
+// RUN: ls %t/cas | wc -l | grep 2
+// RUN: ls %t/cas | grep v1.1
```

Comment:
Do you know if `wc` exists on Windows?

Reply:
It's used in a handful of test files without checking for platform or even requiring shell.